File: YummYummY-frontend/src/app/models/Recipe.ts
export class Recipe {
  IDRecipe: number;
  Name: string;
  MainIngredient: string;
  Nationality: string;
  Author: string;
  constructor(id: number, name: string, mainIngredient: string, nationality: string, author: string) {
    this.IDRecipe = id;
    this.Name = name;
    this.MainIngredient = mainIngredient;
    this.Nationality = nationality;
    this.Author = author;
  }
}

File: Backend-Core 3.1/Models/Recipe.cs
using System;
using System.Collections.Generic;

namespace Backend_Core_3._1.Models
{
    public partial class Recipe
    {
        public Recipe()
        {
            IngredientList = new HashSet<IngredientList>();
            Step = new HashSet<Step>();
        }

        public int Idrecipe { get; set; }
        public string Name { get; set; }
        public int Idingredient { get; set; }
        public int? Idnationality { get; set; }
        public string Author { get; set; }
        public DateTime DtimeCreate { get; set; }
        public TimeSpan? Duration { get; set; }

        public virtual Ingredient IdingredientNavigation { get; set; }
        public virtual Nationality IdnationalityNavigation { get; set; }
        public virtual ICollection<IngredientList> IngredientList { get; set; }
        public virtual ICollection<Step> Step { get; set; }
    }
}
